### ISSUE TITLE
Status 406 when attempting to read metrics from rabbitmq

### ISSUE THREAD

**Issue by CristianPupazan** [kbudde/rabbitmq_exporter#84]
I deployed this via a the latest helm chart on kubernetes. The deployment is successful but I get the following error in the logs:

`time="2018-06-07T15:29:55Z" level=error msg="Error while retrieving data from rabbitHost" error="<nil>" host="http://my-host:15672" statusCode=406`

Is this something to do with the `accept headers` passed to rabbitmq ? Does this work with RabbitMQ 3.6.8, Erlang R16B03 ? 

Thanks

**Comment by kbudde** [comment]
Hi,

which version of rabbitmq and which capabilities are you using?

If your rabbitmq is too old you will see an error 406. Please check #83

**Comment by CristianPupazan** [comment]
Hi,
Thanks for the pointers. I am using RabbitMQ **3.6.8**. I was using both `bert,no_sort` capabilities. So the 406 error was clearly because `bert` only works with 3.6.9 +
`no_sort` should work but for some reason rabbitMQ fails with a 500. There seems to be a bug in this rabbitMQ version. 

I got it working by dropping both capabilities. Might consider upgrading rabbitMQ at a later stage. 

Closing.
